### DISCUSSION TITLE
"Inactive two-step login" duplicates entries with more than one URI

### DISCUSSION THREAD

**Discussion by My-Random-Thoughts** [dani-garcia/vaultwarden#3684] [Q&A] [answered]
### Subject of the issue
I have a login entry that have more than one URI listed.  When I run a report for "Inactive two-step login", that entry is listed once for every URI.  So a login with 4 URIs will be listed 4 times  

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.29.0
* Web-vault version: v2023.5.0
* OS/Arch: linux/x86_64
* Running within Docker: true (Base: Debian)
* Environment settings overridden: true
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.41.2
* Clients used: Web Vault, desktop, Android
* Reverse proxy and version: none
* Other relevant information: 

### Steps to reproduce
- Using the Web Vault, create a new login vault entry for a site that uses 2FA (github.com for example)
- Add more than one URI
- Go to the reports, Inactive two-step login page

### Expected behaviour
- Only one entry should be shown.

### Actual behaviour
- The entry has more than one listing, throwing off the count and confusing users

### Troubleshooting data
Screenshots...
![vw1](https://github.com/dani-garcia/vaultwarden/assets/21334396/5f323609-414d-4943-a149-e15d9e72c3c7)
![vw2](https://github.com/dani-garcia/vaultwarden/assets/21334396/297fe629-d1f8-4e02-8042-b428645f0ba7)

**Comment by BlackDex** [comment, marked as answer]
This is not something we can fix as this is a Web-Vault specific feature.
I suggest to report this upstream at Bitwarden.

**Comment by My-Random-Thoughts** [comment]
I thought it might be, but wanted to check here first.
Thanks
